schema_editor/test/spec/schemas/schemas-service.spec.js

```diff
@@ -148,4 +176,69 @@ describe('ase.schemas:Schemas', function () {
             message: jasmine.stringMatching('Diff')
         }]));
     });
+
+    // NOTE: This test must be updated when new field types are added.
+    it('should preserve all information when serializing and deserializing schemas', function () {
+        // In practice this means that the serialize and deserialize methods are inverses
```

Comment:
@CloudNiner - Yeah, I'm really curious how to pull the json file into a test. I tried several different ways and was unsuccessful, but surely there's a way.